README.md

```diff
@@ -9,91 +9,168 @@ A libGDX cross platform video rendering extension
+implementation "com.badlogicgames.gdx-video:gdx-video-lwjgl3:$gdxVideoVersion"
+// or (when using legacy LWJGL2):
+implementation "com.badlogicgames.gdx-video:gdx-video-lwjgl:$gdxVideoVersion"
 ```
-or
+
+##### Android:
 ```groovy
-implementation "com.badlogicgames.gdx-video:gdx-video-lwjgl3:0.0.1"
+implementation "com.badlogicgames.gdx-video:gdx-video-android:$gdxVideoVersion"
 ```
 
-##### Android:
+##### iOS (with RoboVM):
 ```groovy
-implementation "com.badlogicgames.gdx-video:gdx-video-android:0.0.1"
+implementation "com.badlogicgames.gdx-video:gdx-video-robovm:$gdxVideoVersion"
 ```
 
 ##### Html:
 
 ```groovy
-implementation "com.badlogicgames.gdx-video:gdx-video:0.0.1:sources"
-implementation "com.badlogicgames.gdx-video:gdx-video-gwt:0.0.1"
-implementation "com.badlogicgames.gdx-video:gdx-video-gwt:0.0.1:sources"
+implementation "com.badlogicgames.gdx-video:gdx-video:$gdxVideoVersion:sources"
+implementation "com.badlogicgames.gdx-video:gdx-video-gwt:$gdxVideoVersion"
+implementation "com.badlogicgames.gdx-video:gdx-video-gwt:$gdxVideoVersion:sources"
 ```
 You also need to add the following file to your GdxDefinition.gwt.xml in your html project:
 `````xml
 <inherits name="com.badlogic.gdx.video.gdx_video_gwt" />
 `````
-#### Maven
-Don't forget to replace _0.0.1_ with the current version!
-
-Core:
-```xml
-<dependency>
-  <groupId>com.badlogicgames.gdx-video</groupId>
-  <artifactId>gdx-video</artifactId>
-  <version>0.0.1</version>
-</dependency>
+
+## Playing video
+
+First, get a video player for the current platform using
+
+```java
+VideoPlayer player = VideoPlayerCreator.createVideoPlayer();
 ```
-Desktop:
-```xml
-<dependency>
-  <groupId>com.badlogicgames.gdx-video</groupId>
-  <artifactId>gdx-video-lwjgl</artifactId>
-  <version>0.0.1</version>
-</dependency>
+
+Then, open a video from your game's assets.
+
+```java
+FileHandle file = Gdx.files.internal("video.webm");
+player.play(file);
 ```
-or
-```xml
-<dependency>
-  <groupId>com.badlogicgames.gdx-video</groupId>
-  <artifactId>gdx-video-lwjgl3</artifactId>
-  <version>0.0.1</version>
-</dependency>
+
+The file gets loaded and starts playing as soon as the first frames
+are decoded. If you want to load a video ahead of time, `pause()` 
+immediately and `resume()` playback later.
+
+Once the video has fully loaded, you may retrieve additional
+information about the file.
+
+```java
+if(player.isBuffered()) {
+    int videoWidth = player.getVideoWidth();
+    int videoHeight = player.getVideoHeight();
+}
 ```
-Android:
-```xml
-<dependency>
-  <groupId>com.badlogicgames.gdx-video</groupId>
-  <artifactId>gdx-video-android</artifactId>
-  <version>0.0.1</version>
-</dependency>
+
+On each frame, call the `update()` function to acquire new video frames
+and keep the audio running. You may then retrieve the frame using `getTexture()`,
+but note that the texture may be larger than the video itself.
+
+```java
+class VideoActor extends Actor {
```

Comment:
Maybe we can also provide a `VideoActor` inside a subpackage `scenes/scene2d`?

Reply:
I agree that would be useful. I'd rather make it simple though, with the user providing and controlling the VideoPlayer directly.

Reply:
Done in 7309a98